contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -17,8 +17,11 @@
 
 import requests
 
-logger = logging.getLogger(__name__)
+from azure.core.exceptions import ClientAuthenticationError
+from azure.identity._exceptions import CredentialUnavailableError
 
+logger = logging.getLogger(__name__)
+_MONITOR_OAUTH_SCOPE = "https://monitor.azure.com/.default"
```

Comment:
This should be "https://monitor.azure.com//.default" ? "https://monitor.azure.com/.default" as per the example you have shared will generate a token with "aud": "https://monitor.azure.com", but the expected audience is "https://monitor.azure.com/" (as per Zaki's inputs).